app/components/wu-icon.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line ember/no-classic-components
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { alias } from '@ember/object/computed';
+// import layout from '../templates/components/wu-icon';
+
+// eslint-disable-next-line ember/no-classic-classes
+let WuIconComponent = Component.extend({
```

Comment:
Indien deze component ergens extern gesourced is, graag bron met link vermelden. Vooral naar onderhoud en verdere legacy-refactoring later toe ...

Reply:
Goed opgemerkt, bedankt! Ik pas dit morgen nog even aan 👍

Reply:
Done, is dit voldoende qua info @MikiDi?

Reply:
Perfect. Dank!